src/dashboard/src/components/accounts/views.py

```diff
@@ -53,7 +54,7 @@ def add(request):
             api_key.key = api_key.generate_key()
             api_key.save()
             user_profile = UserProfile.objects.get(user=newuser)
-            userprofileform = UserProfileForm(request.POST, instance=user_profile)
+            userprofileform.instance = user_profile
```

Comment:
I find this approach to set the instance of a model in a existing form interesting. I've always considered Django forms immutable. Do you know if if this safe doing considering the data initialization of the form (required fields, etc) has been already executed before?

Reply:
Hmm you're right, [Django docs](https://docs.djangoproject.com/en/3.0/ref/forms/api/#bound-and-unbound-forms) advise to create a new instance of a form if the data changes. That's an easy fix.